processor/src/test/java/org/mapstruct/ap/test/bugs/_1720/SharedConfig.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.bugs._1720;
+
+import org.mapstruct.MapperConfig;
+import org.mapstruct.Mapping;
+import org.mapstruct.Mappings;
+
+@MapperConfig
+public interface SharedConfig {
+
+    @Mappings(
+            @Mapping(target = "fullName", source = "source.name")
+    )
+    Target map(Source source);
```

Comment:
The other way around.. what would it mean for the mapping annotated method if your configuration would look like:

```java

@Mapping( target = "fullName1", source = "souce1.name" )
@Mapping( target = "fullName2", source = "souce2.name" )
Target map( Source source1, Source source2, int value);

```
and your mapper

```java
 @InheritConfiguration
Target map( Source mySource)
```